ci/README.md

```diff
@@ -15,3 +15,24 @@ Individual release channels/environments are config files that are passed to inv
 ```bash
 poetry run invoke release -f channels/test.yaml
 ```
+
+## GitOps
+
+```mermaid
+flowchart TD
+classDef default fill:white, color:black, stroke:black, stroke-width:1px
+classDef group_labelstyle fill:#cde6ef, color:black, stroke-width:0px
+class ingestion_label,modeling_label,analysis_label group_labelstyle
+
+pr[Push commits to a branch.\nTest environment?]
+candidates_branch[GitHub Action renders candidates/branch-name]
+branch_diff[invoke diff renders on test PR]
+branch_invoke[invoke releases to test]
+
+candidates_main[GitHub Action builds images and renders candidates/main]
+prod_diff[invoke diff renders on prod PR]
+prod_invoke[invoke releases to prod]
+
+pr -- Yes --> candidates_branch -- "Open PR from candidates/branch-name to releases/test" --> branch_diff -- "Merge candidate PR to releases/test" --> branch_invoke -- Merge to main after review and testing --> candidates_main -- "Open PR from candidates/main to releases/prod" --> prod_diff -- "Merge candidate PR to releases/prod" --> prod_invoke
+pr -- "No; merge to main after review" --> candidates_main
+```
```

Comment:
I guess this may be obvious but is it helpful to make some kind of distinction (maybe bullets below or perhaps could change shapes in the mermaid diagram) between code changes and deployments? I.e., merging PRs to main changes code but the GH actions deploy. So if you don't merge `candidates/main` at the end, you will have changed the main codebase but won't have deployed those changes and they won't run anywhere, right?

Reply:
So like, regardless of the test branch situation, you only ever merge your core code changes once right? From your normal git branch to main? But the candidate branch stuff is to deploy changes, either for testing or for real?

Reply:
Correct, yes. I think I addressed this with a clarification